e2e-test/cypress/integration/currency_format_spec.js

```diff
@@ -0,0 +1,47 @@
+
+describe("Checking format for Value and currency Symbol of all languages", function () {
+  let languageArray = [
+    {
+      language: "en-gb",
+      formattedValue: "€ 12,345.00"
+    },
+    {
+      language: "fr",
+      formattedValue: "12.345,00 €"
+    },
+    {
+      language: "pt",
+      formattedValue: "12.345,00 €"
+    },
+    {
+      language: "de",
+      formattedValue: "12.345,00 €"
+    }
+  ];
+  const standardBudget = [
+    {
+      organization: "Test",
+      value: "12345",
+      currencyCode: "EUR"
+    }
+  ];
+
+
+  it(`Log in and out on overview page for each language`, function () {
```

Comment:
describe **what** is tested. 
Suggestion: "The selected language is affecting the currency format"